sass/_header.scss

```diff
@@ -439,9 +439,9 @@ li.search {
       background-color: $nypl-search-color;
       font-size: 0.9rem;
       right: 0;
-      padding: 1px 3px 3px;
+      padding: 0.1875rem;
       position: absolute;
-      top: 2rem;
+      top: 2.65rem;
```

Comment:
this moves the radio buttons far away from the box:
![image](https://cloud.githubusercontent.com/assets/133020/21654303/fdfea0d4-d282-11e6-8a92-a41193574910.png)